scripts/prune_nightly_release_assets.py

```
@@ -63,9 +74,55 @@ def gh_json(*args: str) -> dict:
+    token = github_token()
+    if token:
+        request = urllib.request.Request(
+            github_api_url(path),
+            method=method,
+            headers={
+                "Accept": "application/vnd.github+json",
+                "Authorization": f"Bearer {token}",
+                "User-Agent": "cmux-nightly-prune",
+                "X-GitHub-Api-Version": "2022-11-28",
+            },
+        )
```

Ruff (0.15.17)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)